src/index.ts

```diff
@@ -1,11 +1,17 @@
 
-interface Window {
-  MP_SDK: any;
-}
-
 const showcase = document.getElementById('showcase') as HTMLIFrameElement;
 const key = 'YOUR KEY HERE';
 
+// declare this file is a module
+export {};
```

Comment:
Interesting. I didn't realize that this file didn't export or import anything previously.